src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(String report, String file) {
```

Comment:
The method parameter 'file' should have a more descriptive name, like 'filePath', to clearly indicate its purpose.